<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>890</width>
    <height>809</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>890</width>
    <height>809</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AnixList</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>891</width>
      <height>771</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 63 8pt &quot;Yu Gothic UI Semibold&quot;;</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Главная страница</string>
     </attribute>
     <widget class="QWidget" name="horizontalLayoutWidget_9">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>50</y>
        <width>761</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLineEdit" name="lineEdit_1"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_1">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Найти</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_2">
         <property name="minimumSize">
          <size>
           <width>111</width>
           <height>31</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <item>
          <property name="text">
           <string>По названию</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>По жанру</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>По году выпуска</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Все</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QScrollArea" name="scrollArea">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>110</y>
        <width>761</width>
        <height>571</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>759</width>
         <height>569</height>
        </rect>
       </property>
       <widget class="QTableWidget" name="tableWidget_1">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>761</width>
          <height>571</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>751</width>
          <height>511</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <column>
         <property name="text">
          <string>Название</string>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="background">
          <color>
           <red>230</red>
           <green>230</green>
           <blue>250</blue>
          </color>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Жанр</string>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="background">
          <color>
           <red>230</red>
           <green>230</green>
           <blue>250</blue>
          </color>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Год выпуска</string>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="background">
          <color>
           <red>230</red>
           <green>230</green>
           <blue>250</blue>
          </color>
         </property>
        </column>
       </widget>
      </widget>
     </widget>
     <widget class="QComboBox" name="comboBox_4">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>700</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <item>
       <property name="text">
        <string>Планы</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Просмотрено</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Ожидания</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>700</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Добавить в</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>901</width>
        <height>741</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../Pictures/ttt.jpg</pixmap>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>700</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>700</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <zorder>label</zorder>
     <zorder>horizontalLayoutWidget_9</zorder>
     <zorder>scrollArea</zorder>
     <zorder>comboBox_4</zorder>
     <zorder>pushButton</zorder>
     <zorder>label_5</zorder>
     <zorder>label_6</zorder>
    </widget>
    <widget class="QWidget" name="tab_2">
     <property name="cursor">
      <cursorShape>UpArrowCursor</cursorShape>
     </property>
     <attribute name="title">
      <string>Планы</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>700</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Удалить</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>901</width>
        <height>741</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../Downloads/ttt.jpg</pixmap>
      </property>
     </widget>
     <widget class="QScrollArea" name="scrollArea_3">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>110</y>
        <width>761</width>
        <height>571</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>759</width>
         <height>569</height>
        </rect>
       </property>
       <widget class="QTableWidget" name="tableWidget_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>771</width>
          <height>571</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>751</width>
          <height>511</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <column>
         <property name="text">
          <string>Название</string>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="background">
          <color>
           <red>230</red>
           <green>230</green>
           <blue>250</blue>
          </color>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Жанр</string>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="background">
          <color>
           <red>230</red>
           <green>230</green>
           <blue>250</blue>
          </color>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Год выпуска</string>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="background">
          <color>
           <red>230</red>
           <green>230</green>
           <blue>250</blue>
          </color>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Дата добавления</string>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
        </column>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_11">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>50</y>
        <width>761</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLineEdit" name="lineEdit_2"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Найти</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_3">
         <property name="minimumSize">
          <size>
           <width>111</width>
           <height>31</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <item>
          <property name="text">
           <string>По названию</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>По жанру</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>По году выпуска</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>По дате добавления</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Все</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <zorder>label_2</zorder>
     <zorder>pushButton_6</zorder>
     <zorder>scrollArea_3</zorder>
     <zorder>horizontalLayoutWidget_11</zorder>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Просмотрено</string>
     </attribute>
     <widget class="QWidget" name="horizontalLayoutWidget_14">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>50</y>
        <width>761</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLineEdit" name="lineEdit_3"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_3">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Найти</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_5">
         <property name="minimumSize">
          <size>
           <width>111</width>
           <height>31</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <item>
          <property name="text">
           <string>По названию</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>По жанру</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>По году выпуска</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>По дате добавления</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Все</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>891</width>
        <height>741</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../Downloads/ttt.jpg</pixmap>
      </property>
     </widget>
     <widget class="QScrollArea" name="scrollArea_4">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>110</y>
        <width>761</width>
        <height>571</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_4">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>759</width>
         <height>569</height>
        </rect>
       </property>
       <widget class="QTableWidget" name="tableWidget_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>761</width>
          <height>571</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>751</width>
          <height>511</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <column>
         <property name="text">
          <string>Название</string>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="background">
          <color>
           <red>230</red>
           <green>230</green>
           <blue>250</blue>
          </color>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Жанр</string>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="background">
          <color>
           <red>230</red>
           <green>230</green>
           <blue>250</blue>
          </color>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Год выпуска</string>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="background">
          <color>
           <red>230</red>
           <green>230</green>
           <blue>250</blue>
          </color>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Дата  добавления</string>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
        </column>
       </widget>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>700</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Удалить</string>
      </property>
     </widget>
     <zorder>label_3</zorder>
     <zorder>horizontalLayoutWidget_14</zorder>
     <zorder>scrollArea_4</zorder>
     <zorder>pushButton_7</zorder>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Ожидания</string>
     </attribute>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>891</width>
        <height>741</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../Downloads/ttt.jpg</pixmap>
      </property>
     </widget>
     <widget class="QScrollArea" name="scrollArea_5">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>110</y>
        <width>761</width>
        <height>571</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_5">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>759</width>
         <height>569</height>
        </rect>
       </property>
       <widget class="QTableWidget" name="tableWidget_4">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>761</width>
          <height>571</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>751</width>
          <height>511</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <column>
         <property name="text">
          <string>Название</string>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="background">
          <color>
           <red>230</red>
           <green>230</green>
           <blue>250</blue>
          </color>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Жанр</string>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="background">
          <color>
           <red>230</red>
           <green>230</green>
           <blue>250</blue>
          </color>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Год выпуска</string>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="background">
          <color>
           <red>230</red>
           <green>230</green>
           <blue>250</blue>
          </color>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Новые серии</string>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
        </column>
       </widget>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton_8">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>700</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Удалить</string>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_10">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>50</y>
        <width>761</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLineEdit" name="lineEdit_4"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_4">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Найти</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_6">
         <property name="minimumSize">
          <size>
           <width>111</width>
           <height>31</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <item>
          <property name="text">
           <string>По названию</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>По жанру</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>По году выпуска</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>По дате выхода новых серий</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Все</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>890</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
